src/client/App/useShareDB.ts

```diff
@@ -2,6 +2,15 @@ import { useEffect, useState } from 'react';
 import { randomId } from '../../randomId';
 import { ShareDBDoc, VZCodeContent } from '../../types';
 import { useSubmitOperation } from '../useSubmitOperation';
+import { Request }  from "../utils/Request.js";
+
+// get the docId from the current url request paramter 
+export const getRequestDocId = function(){
```

Comment:
Whoah this is cool! I'd like to learn more about what is going on here. What is the doc id? What is this used for?